tests/Benchmark/ArrayOfBench.php

```diff
@@ -0,0 +1,59 @@
+<?php declare(strict_types = 1);
+
+namespace Orisai\Tests\Benchmark;
```

Comment:
Should be `Tests\Orisai\ObjectMapper\Benchmark`
PHPStorm should be able autocomplete namespaces if you enable composer.json sync feature
Also `make cs` would tell you